src/App.js

```diff
@@ -1,25 +1,35 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes, Link } from 'react-router-dom';
+import './styles/App.css';
+import React from 'react';
+import StoreBook from './components/Bookstore';
+import Category from './components/Status';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <nav className="nav-bar">
+        <h1>Bookstore CMS</h1>
+        <ul className="links">
+          <li>
+            {' '}
+            <Link className="Books" to="/">
+              Books
+            </Link>
+          </li>
+          <li>
+            {' '}
+            <Link className="category" to="/Category">
+              Category
+            </Link>
+          </li>
+        </ul>
+      </nav>
```

Comment:
[ Optional Changes ]

The beauty of React is the component-based architecture. You can build your whole app in conjunction with tiny tiny components.

- [ ] What if you could move this nav code into a separate Navbar.js component and then import the Navbar component here? I think that will make your App.js component lean and clean. Please try it and see how clean your code will be 👍🏽